src/main/java/appeng/api/config/Settings.java

```diff
@@ -41,6 +41,7 @@ public enum Settings {
     IO_DIRECTION(EnumSet.of(RelativeDirection.LEFT, RelativeDirection.RIGHT)),
 
     BLOCK(EnumSet.of(YesNo.YES, YesNo.NO)),
+    SBLOCK(EnumSet.of(YesNo.YES, YesNo.NO)),
```

Comment:
Please, choose names that are easy to understand. No need in *mysterious* abbreviations. For example:
```suggestion
    SMART_BLOCK(EnumSet.of(YesNo.YES, YesNo.NO)),
```